cypress/support/step_definition/account/changeLanguageSteps.spec.js

```diff
@@ -0,0 +1,23 @@
+import Account from '../../pages/account/pageAccount'
+
+const account = new Account()
+
+Then('I can see initials of my name', () => {
+  account.nameInitialsIsVisible()
```

Comment:
here is the same method name to be updated (verb + noun) > ex: seeNameInitials()